Campaign/management/commands/ProcessCampaignData.py

```diff
@@ -40,48 +40,48 @@ def add_arguments(self, parser):
     def handle(self, *args, **options):
         # Identify Campaign instance for given name.
         try:
-            campaign = Campaign.get_campaign_or_raise(
-                options['campaign_name']
-            )
-
+            campaign = Campaign.get_campaign_or_raise(options['campaign_name'])
```

Comment:
What happened to my beloved `black` formatting?

Reply:
I don't think we have formatting with Black setup for Appraise, it was for OCELoT. But I will definitely use it, especially for new files.
In some cases, I probably just combined some longer statements into a single line for greater readability when reading the code, in particular `print` statements. That 75 characters limit was killing me. I will take a look and use Black.
Thanks for the feedback!

Reply:
I have reformated files touched in this pull request using the default line limit in Black. We may have a separate pull request for with all other files cleaned up.